_config.yml

```diff
@@ -125,3 +136,20 @@ transl:
     footer_placeholder_subscribe: your@email.com
     arrow_prev: Previous Tip
     arrow_next: Next Tip
+  zh_CN:
```

Comment:
@loverajoel  I think I have adapt the defaults vars here. If it was anything wrong. Please tell me. :smile: